apisix/plugins/limit-count.lua

```diff
@@ -132,7 +132,9 @@ function _M.check_schema(conf)
     end
 
     if conf.policy == "redis-cluster" then
-        -- TODO
+        if not conf.redis_serv_list then
+            return false, "missing valid redis option serv_list"
```

Comment:
that is a wrong way, we can use JSONSchema to check this

Reply:
updated